extension/src/environment.rs

```diff
@@ -67,6 +68,11 @@ pub(crate) struct Env<'a, 'b, E: pallet_revive::chain_extension::Ext, S: ?Sized
 
 impl<'a, 'b, E: pallet_revive::chain_extension::Ext, S: ?Sized + Memory<E::T>> Environment
 	for Env<'a, 'b, E, S>
+where
+	E: pallet_revive::chain_extension::Ext,
+	E::T: frame_system::Config,
+	<E::T as frame_system::Config>::AccountId: From<AccountId32> + Decode,
```

Comment:
just documenting (jd:): these bounds are unlikely unnecessary if using the AddressMapper trait instead of concrete impl